signalling/simple-server.py

```diff
@@ -46,6 +47,10 @@
 
 ############### Helper functions ###############
 
+async def health_check(path, request_headers):
+    if path == "/health/":
```

Comment:
Will this match just `/health/` or also `/health`?

Reply:
Just the former, though it'd be simple enough to match both.

Reply:
Please make that change and I'll merge it. Thanks!